prototype/requirements.txt

```diff
@@ -1,3 +1,4 @@
 pyramid
 requests
-
+nose
```

Comment:
these should be test requirements right? Or not, it's not split up in prototype yet nvm.